packages/server/src/service/letter/index.ts

```diff
@@ -116,7 +116,7 @@ export class Letter {
           ? `cid:${this.idPhotoAttachment.filename}`
           : this.info.idPhoto,
       },
-    )
+    ).replace(/((\n|( +)\n|\n( +)){3})+/g, '\n')
```

Comment:
OK -- isn't the regex `\n|( +)\n|\n( +))` just `( *)\n( *)`?

Also if instead of `{3}`,I think we want `{3,}` (3 or more) to be slightly more robust.

Can you also document why we're doing this.